docs/content/latest/quick-start/create-local-cluster/macos.md

```diff
@@ -52,9 +52,7 @@ showAsideToc: true
 
 {{< note title="Note" >}}
 
-This macOS Quick Start is based on the new [`yugabyted`](../../../reference/configuration/yugabyted/) server. You can refer to the older [`yb-ctl`](../../../admin/yb-ctl/) based instructions in the [v2.1 docs](/v2.1/quick-start/install/linux/).
-
-Note that yugabyted currently supports creating a single-node cluster only. Ability to create multi-node clusters is under [active development](https://github.com/yugabyte/yugabyte-db/issues/2057). 
+This macOS Quick Start uses the [`yugabyted`](../../../reference/configuration/yugabyted/) server. You can refer to the older [`yb-ctl`](../../../admin/yb-ctl/) based instructions in the [v2.1 docs](/v2.1/quick-start/install/linux/).
```

Comment:
Another spot, we refer to the 2.0 docs. What's the latest version we can refer to for each place?

Reply:
2.1 docker appears to use yugabyted, while 2.0 docker, 2.1 macos and 2.1 linux use ctl-based